content/rc/databases/view-edit-database.md

```diff
@@ -145,6 +146,32 @@ Here, you can:
 
 For best results, we recommend backing up data before starting any **Danger Zone** actions.
 
+## Manage the database list
+
+The **Databases** list summarizes the status of all databases from the subscriptions associated with your account.  
+
+You can:
+
+- Sort the list in descending or ascending order using the the arrow displayed to right of the field name in the header.  Supported fields include _Subscription_, _Name_, and _Memory_.
```

Comment:
Do you want these italicized fields to be bold instead to match the formatting in the next bullet point?
```suggestion
- Sort the list in descending or ascending order using the the arrow displayed to right of the field name in the header.  Supported fields include **Subscription**, **Name**, and **Memory**.
```

Reply:
Fixed.